src/whatsnew.adoc

```diff
@@ -95,95 +102,89 @@ their position in the table (i.e., the `$id` syntax).
-encodings, the Ion 1.0 length prefixed encoding and the new delimited form that end with the `0xAD` end opcode.  Struct
-values have three encodings, the Ion 1.0 length prefixed encoding, the length prefixed encoding using `VarSym` to encode
-field names (allowing for inline symbol text), and the delimited form that encodes its field names with `VarSym` (there
-is no delimited form with Ion 1.0 encoding of field names).  Symbol values have two encodings, one is the Ion 1.0
-encoding with the symbol ID, and the other one is structurally identical to the encoding of strings, which are used for
-symbols with inline text.
+Typed `null` values are now link:binary-encoding.adoc#nulls[encoded in two bytes using the `0xEB` opcode].
 
-Annotated values no longer have an outer length container.  They are now encoded with an opcode that specifies a single
-annotation with value following; an opcode that specifies two annotations with a value following; and finally, an opcode
-that specifies a variable length of annotations followed by a value.  The latter encoding is similar to how Ion 1.0
-annotations are encoded with the exception that there is no outer length.
+Lists and S-expressions have link:binary-encoding.adoc#lists[two encodings]: a length-prefixed encoding and a new
+delimited form that ends with
+the `0xF0` opcode.
+
+Struct values have link:binary-encoding.adoc#structs[three encodings]: a length-prefixed encoding which uses symbol IDs
+for its field names, a length-prefixed encoding which uses `FlexSym` for its field names (allowing for inline symbol text
+as needed), and a delimited form that encodes its field names with `FlexSym` (there is no delimited form with symbol ID
+field names).
+
+Symbol values have link:binary-encoding.adoc#symbols_with_inline_text[two encodings]: one is the Ion 1.0-style
+encoding using the symbol ID, and the other one is structurally identical to the encoding of strings, supplying its
+text's UTF-8 bytes inline.
+
+link:binary-encoding.adoc#annotations[Annotation sequences] are a prefix to the value they decorate, and no longer
+have an outer length container.
+They are now encoded with an opcode that specifies a single annotation with value following, an opcode that specifies
+two annotations with a value following, and finally, an opcode that specifies a variable length of annotations followed
+by a value.
+The latter encoding is similar to how Ion 1.0 annotations are encoded with the exception that there is no
+outer length.
 
 IMPORTANT: *_Discussion_*: Should we provide an op-code for length prefixing the entire annotation?  If so, where should
 it go? E.g, make the variable length SID based annotations support this.
 
-Integers now use an `Int` sub-field instead of the Ion 1.0 encoding using sign magnitude (with two opcodes).
+link:binary-encoding.adoc#integers[Integers] now use an `Int` sub-field instead of the Ion 1.0 encoding using sign-and-magnitude
```

Comment:
```suggestion
link:binary-encoding.adoc#integers[Integers] now use a `FixedInt` sub-field instead of the Ion 1.0 encoding that used sign-and-magnitude
```